graylog2-web-interface/src/components/indices/IndexSetTemplates/TemplateDetails.tsx

```diff
@@ -74,66 +83,98 @@ const formatRefreshInterval = (intervalInMs : number) => {
 
```

Comment:
We should add an Edit button to the template details page.
![image](https://github.com/Graylog2/graylog2-server/assets/1381455/e2e29516-daf0-47b0-aa44-5ae25e5ce68e)